### DISCUSSION TITLE
Error while running the "Sunbird-org create" in postman configurarions.

### DISCUSSION THREAD

**Discussion by Arunkumarpodili** [Sunbird-Ed/Community#672] [Installation]
Hi @swayangjit,

   We are installing Sunbird 6.0.0 in our environment, while doing the Postman configurations we are encountering the error PFA below screenshot, we need your help to resolve this issue, if possible we can have a 10-minute quick call to discuss this in detail. 
you can send meeting invites for the below mail IDs. 
arunkumar.p@optit.in
rajasimman.s@optit.in 

error:
![image](https://github.com/Sunbird-Ed/Community/assets/121008660/6ef198bd-1252-432f-b4e3-d138321ea736)

**Comment by gandham-santhosh** [comment]
@Arunkumarpodili did you check if there are any errors from the user org service?

**Comment by anisha147** [comment]
Hi @Arunkumarpodili 

We hope you were able to resolve the issues after checking the comment from @gandham-santhosh. If yes, could you please close this thread from your end?

cc: @gandham-santhosh @swayangjit
